src/mbgl/layout/symbol_layout.cpp

```diff
@@ -90,10 +92,12 @@ SymbolLayout::SymbolLayout(std::string bucketName_,
                 u8string = platform::lowercase(u8string);
             }
 
-            ft.text = util::utf8_to_utf32::convert(u8string);
+            std::u16string u16string = util::utf8_to_utf16::convert( u8string );
```

Comment:
I ran clang-tidy and clang-format on all the files, and it looks like they hadn't been tidied in a while, so it introduced a lot of diffs unrelated to my changes. Should I still go ahead with the changes?